src/RequestHandler.cpp

```diff
@@ -43,42 +42,37 @@ void RequestHandler::checkMetadataModeArguments(
     showTopicPartitionOffsets(UserArguments);
 }
 
-std::string RequestHandler::subscribeConsumeAtOffset(std::string TopicName,
-                                                     int64_t Offset) {
-  int64_t EOFPartitionCounter = 0,
-          NumberOfPartitions =
-              KafkaConnection->getNumberOfTopicPartitions(TopicName);
-  std::pair<std::string, bool> MessageAndEOF;
-  int i = 0;
-  // SUBSCRIBE AT AN OFFSET
+void RequestHandler::subscribeConsumeAtOffset(std::string TopicName,
+                                              int64_t Offset) {
+  int EOFPartitionCounter = 0,
+      NumberOfPartitions =
```

Comment:
I think this is more readable as two separate declarations. Also in this case `getNumberOfTopicPartitions` returns an `int64_t` not an `int`.